github-upload-secrets

```diff
@@ -52,7 +51,7 @@ def encrypt(public_key: str, secret_value: str) -> str:
     return b64encode(encrypted).decode("utf-8")
 
 
-def generate_ssh_key(pubkey):
+def generate_ssh_key():
     """Generates a Ed25519 keypair and outputs public/private in OpenSSH format"""
     """We could also do this, using cryptography.hazmat.primitives:
```

Comment:
BTW, it's technically valid, but not really semantically legit to have two subsequent docstrings. The second one should be a comment.

Reply:
I always considered this to be a matter of taste, but after a bit of reading it seems that most sources agree that non-docstring block comments are better done with `#` on each line.